docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -0,0 +1,170 @@
+---
+id: zeebe-backup-and-restore
+title: "Zeebe Backups"
+description: "A guide to creating backup of a running Zeebe cluster."
+keywords: ["backup", "backups"]
+---
+
+:::note
+This is an experimental feature in 8.1.0. The APIs might change in later releases.
```

Comment:
I propose to make this even more transparent. If it is experimental, people should not rely on it to make backups, imho.
For Web Modeler Self-Managed Beta, for instance, we use this disclaimers:

> It is not recommended for production use, and there is no maintenance service guaranteed. Special [terms & conditions](https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/) apply.

See
https://docs.google.com/document/d/1G3VOJfvtDxSf4kk5YM072XELZK6tBCtxkrop76tyl80/edit#heading=h.iqjs32b4jjoa

We'll introduce the terms "alpha" and "beta" here to avoid ambiguity:  https://github.com/camunda/camunda-platform-docs/pull/1304 

Linking to that docs page could be helpful then

Reply:
@christian-konrad I discussed this with @felix-mueller . We would like to offer this feature with production support. The only disclaimer is that API could change in the next release. We do not want to tag it as beta because then we will see no adoption and probably almost no feedback.  I can remove "experimental feature" from the disclaimer. Is that ok?

Reply:
Perfect!